sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -23,3 +37,482 @@ var client = new TimeSeriesInsightsClient(
     tokenCredential);
 ```
 
+## Create, get, replace, and delete instances
+
+### Create instances
+You can use the following snippet to create instance Id using `TimeSeriesId`.
+
+```csharp
+TimeSeriesId instanceId = modelSettings.TimeSeriesIdProperties.Count switch
```

Comment:
In this snippet, modelSettings is unknown. Maybe in the paragraph before this snippet, you can explain that you can use the Model Settings client to learn more about how the Time Series Id is composed of in the environment.

Reply:
Sure will update it.